components/script/dom/servoparser/async_html.rs

```diff
@@ -4,12 +4,13 @@
 
 #![allow(unrooted_must_root)]
 
+use core::iter::FromIterator;
+use core::str::FromStr;
```

Comment:
Aren't these already imported? At the very least, you shouldn't depend on them from `core`.